Tools/CISupport/ews-build/steps.py

```diff
@@ -3776,6 +3790,9 @@ def setLayoutTestCommand(self):
         patch_author = self.getProperty('patch_author')
         if patch_author in ['webkit-wpt-import-bot@igalia.com']:
             self.setCommand(self.command + ['imported/w3c/web-platform-tests'])
+        elif GitHub.NO_FAILURE_LIMITS in self.getProperty('labels', []):
+            self.setCommand(self.command + ['--no-retry'])
+            self.maxTime = 60 * 90
```

Comment:
Please verify if maxTime works fine in buildbot while set in this manner, some of the variables (like timeout) need to be explicitly passed in __init__. And in the only place where we currently use maxTime in this code, we pass it in __init__.

Reply:
`maxTime` will not work when set in this manner, based on code inspection. We will need to override `buildCommandKwargs`.